src/Tribe/Repositories/Attendee/Commerce.php

```diff
@@ -77,4 +115,149 @@ public function checked_in_keys() {
+
+			$args['price_currency'] = $currency_symbol;
+		}
+
+		return $args;
+	}
+
+	/**
+	 * Handle backwards compatible actions for Tribe Commerce.
+	 *
+	 * @since TBD
+	 *
+	 * @param WP_Post                       $attendee      The attendee object.
+	 * @param array                         $attendee_data List of additional attendee data.
+	 * @param Tribe__Tickets__Ticket_Object $ticket        The ticket object.
+	 */
+	public function trigger_create_actions( $attendee, $attendee_data, $ticket ) {
+		parent::trigger_create_actions( $attendee, $attendee_data, $ticket );
+
+		$attendee_id           = $attendee->ID;
+		$post_id               = Arr::get( $attendee_data, 'post_id' );
+		$order_id              = Arr::get( $attendee_data, 'order_id' );
+		$product_id            = $ticket->ID;
+		$order_attendee_id     = Arr::get( $attendee_data, 'order_attendee_id' );
+		$attendee_order_status = $attendee_data['attendee_status'];
+
+		/**
+		 * Action fired when an PayPal attendee ticket is created
+		 *
+		 * @since 4.7
+		 *
+		 * @param int    $attendee_id           Attendee post ID
+		 * @param string $order_id              PayPal Order ID
+		 * @param int    $product_id            PayPal ticket post ID
+		 * @param int    $order_attendee_id     Attendee number in submitted order
+		 * @param string $attendee_order_status The order status for the attendee.
+		 */
+		do_action( 'event_tickets_tpp_attendee_created', $attendee_id, $order_id, $product_id, $order_attendee_id, $attendee_order_status );
+
+		/**
+		 * Action fired when an PayPal attendee ticket is updated.
+		 *
+		 * This action will fire both when the attendee is created and
+		 * when the attendee is updated.
+		 * Hook into the `event_tickets_tpp_attendee_created` action to
+		 * only act on the attendee creation.
+		 *
+		 * @since 4.7
+		 *
+		 * @param int    $attendee_id           Attendee post ID
+		 * @param string $order_id              PayPal Order ID
+		 * @param int    $product_id            PayPal ticket post ID
+		 * @param int    $order_attendee_id     Attendee number in submitted order
+		 * @param string $attendee_order_status The order status for the attendee.
+		 */
+		do_action( 'event_tickets_tpp_attendee_updated', $attendee_id, $order_id, $product_id, $order_attendee_id, $attendee_order_status );
+
+		if ( $post_id ) {
+			$global_stock    = new Tribe__Tickets__Global_Stock( $post_id );
+			$shared_capacity = false;
+
+			if ( $global_stock->is_enabled() ) {
+				$shared_capacity = true;
+			}
+
+			switch ( $attendee_order_status ) {
+				case Tribe__Tickets__Commerce__PayPal__Stati::$completed:
+					$this->attendee_provider->increase_ticket_sales_by( $product_id, 1, $shared_capacity, $global_stock );
+					break;
+				case Tribe__Tickets__Commerce__PayPal__Stati::$refunded:
+					$this->attendee_provider->decrease_ticket_sales_by( $product_id, 1, $shared_capacity, $global_stock );
+					break;
+				default:
+					break;
+			}
+		}
```

Comment:
Wonder if this piece should be added to `event_tickets_tpp_attendee_updated` itself.

Reply:
It could but this is here after moving it out of the PayPal main class where it sets up the attendee. It wasn't attached to the action before and I didn't see the need to attach it in this class separately. Do you think there's a need for that?